jujugui/static/gui/src/app/views/bundle-import-notifications.js

```diff
@@ -38,6 +38,13 @@ YUI.add('bundle-import-notifications', function(Y) {
     watchAll: function(env, db) {
       env.deployerStatus(function(response) {
         if (response.err) {
+          // This is expected in the cases the GUI is not served by the GUI
+          // server, in which case the deployer method is not implemented.
+          if (response.err.indexOf('not implemented') > -1) {
+            console.log(
```

Comment:
A neat thing either @hatched or @huwshimi showed me--you can do console.error for something like this as well, which browsers handle slightly different (e.g. firefox dumps a little more info about where the error came from).